src/python/dxpy/bindings/dxproject.py

```diff
@@ -296,7 +296,7 @@ def new(self, name, summary=None, description=None, protected=None,
         :type protected: boolean
         :param restricted: If provided, whether the project should be restricted
         :type restricted: boolean
-        :param download_restricted: If provided, whether external downloads should be restricted
+        :param download_restricted: If provided, whether external downloads should be restricted. For database objects, whether the data in the project can be accessed by users from outside DNAnexus
```

Comment:
```suggestion
        :param download_restricted: If provided, whether external file downloads and external reads of database object should be restricted.
```

Reply:
I like that. @ptn24 what do you think?

Reply:
I would distinguish database objects because it would be reads and writes. I would go with what you had before, or something like "For database objects, whether external access should be prohibited"